ckanext/pages/i18n/uk_UA/LC_MESSAGES/ckanext-pages.po

```diff
@@ -0,0 +1,287 @@
+msgid "Visibility"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:79
+msgid "Private"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:79
+msgid "Public"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:86
+msgid "Image URL"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:88
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:90
+msgid "Lang"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:109
+msgid "Menu Order"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:112
+msgid "Not in Menu"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:122
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:126
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:130
+msgid "Content"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:122
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:130
+msgid "Enter content here"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:128
+msgid "My content"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:135
+#: ckanext/pages/theme/templates_main/ckanext_pages/confirm_delete.html:15
+msgid "Cancel"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:136
+msgid "Add"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:141
+msgid "Are you sure you want to delete this Page?"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:142
+msgid "Delete"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base_form.html:146
+msgid "Save"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog.html:8
+#: ckanext/pages/theme/templates_main/ckanext_pages/page.html:8
+msgid "Edit"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog.html:27
+#: ckanext/pages/theme/templates_main/ckanext_pages/group_page.html:18
+#: ckanext/pages/theme/templates_main/ckanext_pages/organization_page.html:18
+#: ckanext/pages/theme/templates_main/ckanext_pages/page.html:23
+#: ckanext/pages/theme/templates_main/ckanext_pages/snippets/pages_list.html:51
+#: ckanext/pages/theme/templates_main/ckanext_pages/snippets/pages_list.html:71
+msgid "This page currently has no content"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog.html:54
+msgid "More"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog_edit.html:3
+msgid "Edit Article"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog_edit_base.html:3
+msgid "Blog Articles"
+msgstr ""
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog_edit_base.html:12
+msgid " <p>Add blog articles to CKAN here.</p> "
```

Comment:
Is it ok to have HTML tags in transletion?

Reply:
fixed